### PR TITLE
DDF-04485 fixing styling regression

### PR DESCRIPTION
**Pull request by gjvera**
#### What does this PR do?
Fix styling regression that was a result of #4503 
#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler 
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4485 

#### Screenshots
regression
image
fix
image (1)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by gjvera** [comment]
build now

**Comment by bdeining** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/visualization-selector/visualization-selector.js [hunk 1]
```diff
@@ -23,17 +23,17 @@ const CustomElement = styled.div`
 const Visualization = styled.div`
   cursor: pointer;
   opacity: ${props => props.theme.minimumOpacity}
+  padding: ${props => props.theme.largeSpacing}
   :hover {
     opacity: 1
   }
 `
 const VisualizationChoice = styled.div`
```
Comment (andrewkfiedler) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/visualization-selector/visualization-selector.js]: ❗️ VisualizationChoice appears to be unused, so these styles should move up into Visualization.
```suggestion
const Visualization = styled.div`
  cursor: pointer;
  opacity: ${props => props.theme.minimumOpacity};
  padding: ${props => props.theme.largeSpacing};
  :hover {
    opacity: 1;
  }
  white-space: nowrap;
  cursor: move;
  cursor: grab;
  cursor: -moz-grab;
  cursor: -webkit-grab;
`
```

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/visualization-selector/visualization-selector.js [hunk 2]
```diff
@@ -23,17 +23,29 @@ const CustomElement = styled.div`
...
+  :hover {
+    opacity: 1;
+  }
+  white-space: nowrap;
+  cursor: move;
+  cursor: grab;
+  cursor: -moz-grab;
+  cursor: -webkit-grab;
+`
+  white-space: nowrap;
```
Comment (andrewkfiedler) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/visualization-selector/visualization-selector.js]: Looks like accepting my change orphaned this block.  Not really sure how suggestions work 🤔

**Comment by andrewkfiedler** [comment]
Verified in hero, verified local build works.

**Pull request merged**